standard/i18n_data.lua

```diff
@@ -0,0 +1,13 @@
+---
```

Comment:
assume the other needed data gets added in sep. PRs

Reply:
Correct, just added enough data for test